index.html

```diff
@@ -8,44 +8,275 @@
+                        <h2 class="works-item-title"> Tonic </h2>
+                        <div class="works-item-project-info-frame">
+                            <p class="works-item-project-info-canopy works-item-project-info-items"> 
+                                <span>CANOPY</span> 
+                            </p>
+                            <img src="img/Counter.png" alt="..." class="works-item-project-info-point-1 works-item-project-info-items">
+                            <p class="works-item-project-info-type"> 
+                                <span> Back End Dev</span> 
+                            </p>
+                            <img src="img/Counter.png" alt="..." class="works-item-project-info-point-2 works-item-project-info-items">
+                            <p class="works-item-project-info-year"> 
+                                <span>2022</span> 
+                            </p>
+                        </div>
+                    </div>
+                    <p class="works-item-project-description"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="works-item-project-tecnologies">
+                        <li class="works-item-project-tecnologies-item works-item-project-item-html"> 
+                            <span> html </span> 
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-css"> 
+                            <span> css </span>
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-javaScript"> 
+                            <span> javaScript </span>
+                        </li>
+                    </ul>
+                    <div id="works-item-project-action">
+                        <button class="works-item-project-action-button">
+                            <span>  See Project </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="about">
+            <div id="top">
+                <h2 class="about-title">About Myself</h2>
+                <p class="about-text"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="about-social-media">
+                    <p class="supporting-text"> 
+                        <a>LET'S CONNECT</a>
+                    </p>
+                    <ul class="about-social-media-links">
+                        <li>
+                            <a href="#">
+                                <img src="./img/twitter.png" alt="twitter link">
+                            </a>
+                        </li>
+                        <li>
```

Comment:
The social media icons are coming to the right side of the button, please make sure to fix it.

![image](https://user-images.githubusercontent.com/34779928/192084599-7c0cd6a2-c38d-4ce5-8219-e1b508b64861.png)